docs/superpowers/specs/2026-05-01-contact-form-design.md

```
@@ -0,0 +1,446 @@
+| 項目 | 決定 |
+|---|---|
+| 送信先 | Resend → `miyata09x0084@gmail.com` |
+| 配置 | `/contact` 専用ページ。トップは導線リンクのみ |
+| 入力項目 | 名前 / メール / 本文（3項目） |
+| スパム対策 | Cloudflare Turnstile |
+| 完了UX | 同一ページ内でフォーム→完了メッセージ差し替え |
+| 言語 | ラベルは pixel 大文字英語、説明文は日本語 |
+
+## アーキテクチャ
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)